digiroad2-oracle/src/main/resources/db/migration/V199__add_new_table_for_traffic_manager_process.sql

```diff
@@ -2,6 +2,7 @@ CREATE TABLE	TRAFFIC_SIGN_MANAGER
 (
```

Comment:
This file needs to be deleted, and this code should be moved to DROTH-1814_vehicle_restrictions_and_traffic_signs_V2